openvpn/common/platform_string.hpp

```diff
@@ -37,6 +37,9 @@ namespace openvpn {
     if (!app_version.empty())
       os << app_version << '/';
     os << OPENVPN_VERSION;
+#if defined GIT_VERSION_STRING
```

Comment:
for the future: we normally use "defined(X)" and not "defined X", like it's done a few lines below.